### PR TITLE
Added pagination for folders.

### PR DESCRIPTION
**Pull request by parrfolio**
Pagination for large folders! Would love to have this.

### CODE REVIEW

**Comment by aknorw** [comment]
Thanks for the PR, did not think about the pagination for the endpoint... Would you mind writing the associated tests ?

**Comment by aknorw** [comment]
I should have asked to add tests in this PR instead of create a new one 😅
Please review my comments in the other PR

File: src/users/collection/getItemsInFolderForUser.test.js [hunk 1]
```diff
@@ -22,8 +22,13 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     })
   })
   it('should get items in a folder in a user\'s collection from its username and a `folderId`', async () => {
-    const data = await client.getItemsInFolderForUser(username, folderId)
+
```
Comment (aknorw) [inline src/users/collection/getItemsInFolderForUser.test.js]: Please do not change that test, and add a new test at the end of the file (`should accept pagination`)

File: src/users/collection/getItemsInFolderForUser.test.js [hunk 2]
```diff
@@ -45,4 +50,18 @@ describe('Users - Collection - getItemsInFolderForUser', () => {
     sort.should.be.equal(by)
     sort_order.should.be.equal(order)
   })
+  it('should accept pagination', async () => {
+    const page = 1
+    const perPage = 3
+    const data = await client.getItemsInFolderForUser({
```
Comment (aknorw) [inline src/users/collection/getItemsInFolderForUser.test.js]: This won't work as you are passing an object to the function. You should have wrote `const data = await client.getItemsInFolderForUser(username, folderId, { page, perPage })`

File: src/users/collection/getItemsInFolderForUser.test.js (hunk 1 above)
Comment (parrfolio) [reply src/users/collection/getItemsInFolderForUser.test.js]: Done

File: src/users/collection/getItemsInFolderForUser.test.js (hunk 2 above)
Comment (parrfolio) [reply src/users/collection/getItemsInFolderForUser.test.js]: Done

**Comment by parrfolio** [comment]
Can we get this patch merged?

**Comment by parrfolio** [comment]
@aknorw can we get this approved?

**Comment by parrfolio** [comment]
Thank you!

**Pull request merged**